content/en/blog/sysflow-falco-sidekick.md

```diff
@@ -0,0 +1,146 @@
+
+As a benefit of using the Sysflow format, users go from managing individual events generated from different sources to obtaining an enhanced dataset to work with, enabling them to focus on writing and sharing analytics on a scalable, common open-source platform.
+
+The framework builds on [Falco libs](https://github.com/falcosecurity/libs) and the [Falco rules language](https://falco.org/docs/rules/) to create the plumbing required for system telemetry as shown in the diagram below.
+
+Falco libs facilitate the collection of system events generated from the host and containers through the eBPF probe in the kernel space.
+
+At user level, The SysFlow Collector, also leveraging Falco libs, collects the previous events and exports the captured information using Apache Avro object serialization
+
+The SysFlow Processor processes and enriches the data. The policy engine applies logical rules to filter, alert or semantically label sysflow records using a declarative language based on Falco rules.
+
+<br/>
+
+![image description](/img/sysflow-falco-sidekick/02-sysflow-falco-libs.png)
+
+<br/>
+
+The SysFlow framework is designed as a pluggable edge processing architecture, which includes a policy engine, and what's more noteworthy, an experimental graph engine.
+
+The policy engine accepts declarative [policies](https://sysflow.readthedocs.io/en/latest/processor.html#writing-runtime-policies) (written in Falco rules syntax) and supports edge filtering, tagging, and alerting on SysFlow streams.
+
+And a really flashy addition is the experimental graph engine, that clusters and exports concise behavioral profiles of monitored applications. We'll see an example in a moment.
+
+Beyond the built-in plugins and exporters provided in the SysFlow stack, users can write and [plug](https://sysflow.readthedocs.io/en/latest/processor.html#write-a-simple-processing-plugin) their own real-time analytics, and consume new telemetry sources using our Golang APIs. The framework also includes Python packages and a pre-built [Jupyter container](https://hub.docker.com/r/sysflowtelemetry/sfnb) to facilitate interactive SysFlow data exploration.
+
+
+### Falco Sidekick integration
+
+[Falco Sidekick](https://github.com/falcosecurity/falcosidekick) is a relatively young project but has already harnessed  3.7M pulls on Docker Hub!
+
+At its core, it's a simple daemon for connecting Falco events to many different ecosystems, like a Swiss Army Knife of sorts. It also provides an intuitive web user interface for event visualization and filtering. 
+
+To demonstrate how SysFlow can extend Falco's capabilities, we have been working on an experimental integration with Falco Sidekick.
+
+The complete ecosystem includes:
+
+* The SysFlow telemetry stack with a new exporter encoder for Falco events.
+* A searchable back store based on S3
+* And Apache Druid, and new extensions to Falco Sidekick for visualizing and interactively querying SysFlow process graphlets.
```

Comment:
Apache Druid, and new extensions to Falco Sidekick for visualizing and interactively querying SysFlow process graphlets.